packages/services/f-braze-adapter/src/utils/areCookiesPermitted.js

```diff
@@ -0,0 +1,10 @@
+import CookieHelper from 'js-cookie';
+
+export const consentCookieName = 'je-cookieConsent';
+export const consentCookieValue = 'full';
+
+export const legacyBannerCookieName = 'je-banner_cookie';
+export const legacyBannerCookieValue = '130315';
```

Comment:
Why `legacyBannerCookieValue` is 130315?

Reply:
Because of [this](https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue#L241) and another internal reference that I'll send to you - It appears it was meant to be a date or version, indicating the latest version of the policy that a user has agreed to.